packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1213,6 +1216,12 @@ function commitWork(current: Fiber | null, finishedWork: Fiber): void {
     case IncompleteClassComponent: {
       return;
     }
+    case EventTarget: {
+      const newProps = finishedWork.memoizedProps;
+      const type = finishedWork.type.type;
+      handleEventTarget(type, newProps, finishedWork);
```

Comment:
This block should be wrapped in the feature flag check so that it can remove everything including the property access etc. That'll also ensure DCE of the function itself once it has more content.

Reply:
I was going to do this in a follow up PR anyway. I can put in a other PR to do this sooner. :)